WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -199,7 +199,8 @@ public boolean addQuickPressShortcut(int blogId, String name) {
     }
 
     /*
-     * used during development to copy database to SD card so we can access it via DDMS
+     * used during development to copy database to an SD card
```

Comment:
The original was fine and concise enough. We should also know what DDMS is and don't need to define it here.